apps/examples/testcase/le_tc/kernel/tc_environ.c

```diff
@@ -54,23 +54,23 @@ static void tc_environ_setenv_getenv_unsetenv(void)
 	char *psz_getvalue = NULL;
 
 	ret_chk = setenv(psz_name, psz_value, 1);
-	TC_ASSERT_EQ_CLEANUP("setenv", ret_chk, OK, clearenv());
+	TC_ASSERT_EQ("setenv", ret_chk, OK);
 
 	psz_getvalue = getenv(psz_name);
-	TC_ASSERT_NEQ_CLEANUP("getenv", psz_getvalue, NULL, clearenv());
-	TC_ASSERT_EQ_CLEANUP("getenv", strcmp(psz_getvalue, psz_value), 0, clearenv());
+	TC_ASSERT_NEQ_CLEANUP("getenv", psz_getvalue, NULL, unsetenv(psz_name));
+	TC_ASSERT_EQ_CLEANUP("getenv", strcmp(psz_getvalue, psz_value), 0, unsetenv(psz_name));
 
 	/* with overwrite_num = 0, psz_value should not be updated */
 
 	psz_value = "pqr";
 	ret_chk = setenv(psz_name, psz_value, 0);
-	TC_ASSERT_EQ_CLEANUP("setenv", ret_chk, OK, clearenv());
+	TC_ASSERT_EQ("setenv", ret_chk, OK);
```

Comment:
I think this line should be
`TC_ASSERT_EQ_CLEANUP("setenv", ret_chk, OK, unsetenv(psz_name));`
because previous setenv still remains